server/repository/src/db_diesel/temperature_breach.rs

```diff
@@ -60,7 +60,7 @@ impl<'a> TemperatureBreachRepository<'a> {
     }
 
     pub fn count(&self, filter: Option<TemperatureBreachFilter>) -> Result<i64, RepositoryError> {
-        let query = Self::create_filtered_query(filter)?;
+        let query = Self::create_filtered_query(filter);
```

Comment:
While exposing create_filtere_query in temperature_log realised that it was returning result (not needed), and followed the string to make other create_filtered_query resultless.

Btw exposing create_filtered_query was needed to filter aggregated charts by existing temperature_log filter